src/freenet/node/PeerManager.java

```diff
@@ -252,6 +252,7 @@ private boolean readPeers(File peersFile, NodeCrypto crypto, OpennetManager open
 				// Read a single NodePeer
 				SimpleFieldSet fs;
 				fs = new SimpleFieldSet(br, false, true);
+				boolean broken = true;
```

Comment:
This is never set to `false`. Did you intend `broken = false;` somewhere in the `try {}`?